### PR TITLE
Fix infinite specific funding target callout

### PR DESCRIPTION
**Pull request by wraeth-eth**
## What does this PR do and why?

_Provide a description of what this PR does. Link to any relevant GitHub issues, Notion tasks or Discord discussions._

## Screenshots or screen recordings

_If applicable, provide screenshots or screen recordings to demonstrate the changes._

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

**Comment by wraeth-eth** [comment]
Current dependencies on/for this PR:
* main
  * **PR #2348** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2348" target="_blank">Graphite</a> 
    * **PR #2349** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2349" target="_blank">Graphite</a> 
      * **PR #2350** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2350" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2350?utm_source=stack-comment).

**Comment by wraeth-eth** [comment]
**Graphite Merge Job**

Current status: ⏳ Queued to merge

This pull request is currently queued to be merged as part of a stack.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2350?utm_source=stack-comment).


Job Reference: FCO7UxXsqYOBvxApXFkP

File: src/components/Create/components/pages/FundingTarget/FundingTargetPage.tsx
Comment (tomquirk) [inline src/components/Create/components/pages/FundingTarget/FundingTargetPage.tsx:134]:
```suggestion
      calloutText = t`An Infinite Funding Target means contributors cannot redeem their tokens for a portion of the treasury funds. Payout splits will can only be nominated in percentages.`
```

File: src/components/Create/components/pages/FundingTarget/FundingTargetPage.tsx
Comment (tomquirk) [inline src/components/Create/components/pages/FundingTarget/FundingTargetPage.tsx:137]:
```suggestion
      calloutText = t`Your Funding Target can be changed in future funding cycles. Payouts denominated in USD are paid out in ETH at the current exchange rate.`
```

**Review by tomquirk — APPROVED** [review]

**Comment by wraeth-eth** [comment]
**Graphite Merge Job**

Current status: 🔄 Waiting for checks to pass

This pull request is currently waiting for CI and mergeability checks to pass before being merged.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2350?utm_source=stack-comment).


Job Reference: tq0DBErRQ758wBw1xKNP

**Pull request merged**
